cli/src/cmd_net.rs

```diff
@@ -116,10 +116,14 @@ impl AuthenticatedCmd for CmdLinkAdd {
             fec: self.fec,
             mtu: self.mtu,
             speed: self.speed,
-            //TODO not fully plumbed on the back end yet.
-            lldp: LldpServiceConfigCreate {
+            lldp: LldpLinkConfigCreate {
```

Comment:
perhaps `lldp` doesn't need to be a required field?

Reply:
No, it doesn't.  We can easily construct a disabled `Default` on the server side.